cmake/ActsCompilerOptions.cmake

```diff
@@ -10,6 +10,11 @@ set(ACTS_CXX_FLAGS_MINSIZEREL "")
 set(ACTS_CXX_FLAGS_RELEASE "")
 set(ACTS_CXX_FLAGS_RELWITHDEBINFO "")
 
+set(ACTS_CXX_STANDARD_FEATURE cxx_std_17)
+if(${CMAKE_CXX_STANDARD} GREATER 17)
+  set(ACTS_CXX_STANDARD_FEATURE "cxx_std_${CMAKE_CXX_STANDARD}")
```

Comment:
Actually I would send a `message(ERROR "...")` in case the user provides `CMAKE_CXX_STANDARD=14` or less and not silently go to C++17

Reply:
Done.